js/data.js

```diff
@@ -0,0 +1,34 @@
+import { randomNumber } from './util';
```

Comment:
Тут не хватает расширения импортируемого файла
```suggestion
import { randomNumber } from './util.js';
```